packages/web-components/src/components/ic-checkbox-group/ic-checkbox-group.types.ts

```diff
@@ -1,3 +1,3 @@
 export interface IcChangeEventDetail {
-  value: string[];
+  checkedOptions: HTMLIcCheckboxElement[];
```

Comment:
This is a breaking change as someone _might_ be using `value`. Appreciate it needs updating so the unchecked value is removed from the array.
Could we not add checkedOptions as a new object emitted from the event?